ext/php8/ddtrace.c

```diff
@@ -67,6 +68,10 @@ STD_PHP_INI_ENTRY("ddtrace.request_init_hook", "@php_dir@/datadog_trace/bridge/d
 STD_PHP_INI_ENTRY("ddtrace.request_init_hook", "", PHP_INI_SYSTEM, OnUpdateString, request_init_hook,
                   zend_ddtrace_globals, ddtrace_globals)
 #endif
+
+// Exposed for testing only
+STD_PHP_INI_ENTRY("ddtrace.test_cgroup_file", "", PHP_INI_SYSTEM, OnUpdateString, test_cgroup_file,
+                  zend_ddtrace_globals, ddtrace_globals)
```

Comment:
```suggestion
// Exposed for testing only
STD_PHP_INI_ENTRY("ddtrace.cgroup_file", "/proc/self/cgroup", PHP_INI_SYSTEM, OnUpdateString, cgroup_file,
                  zend_ddtrace_globals, ddtrace_globals)
```

I don't particularly love it, but I do think it is an improvement because the default value is now passed in, instead of hard-coded into the implementation.

Reply:
I agree this is an improvement. Hopefully we'll have a better option for testing this from the embed SAPI once we get that infrastructure set up.